packages/tasks/src/model-libraries.ts

```diff
@@ -422,6 +422,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		repoUrl: "https://github.com/TensorSpeech/TensorFlowTTS",
 		snippets: snippets.tensorflowtts,
 	},
+	"tic-clip": {
+		prettyLabel: "tic-clip",
```

Comment:
As in https://github.com/apple/ml-tic-clip

```suggestion
		prettyLabel: "TiC-CLIP",
```